.github/workflows/node_build.yml

```diff
@@ -68,3 +57,10 @@ jobs:
           build/
           dist/
```

Comment:
mmh, think we'd need to access the dist and build in later steps?  

are caches the only way access these in later steps? (if so we wouldn't want to cache them solely based on package.json)